packages/editor/src/components/post-author/check.js

```diff
@@ -21,11 +16,7 @@ export default function PostAuthorCheck( { children } ) {
 		const post = select( editorStore ).getCurrentPost();
 		const authors = select( coreStore ).getUsers( AUTHORS_QUERY );
 		return {
-			hasAssignAuthorAction: get(
```

Comment:
`getCurrentPost()` will always be an object:

https://github.com/WordPress/gutenberg/blob/4944315cac156c78313121657dff9d58ce8f4d3f/packages/editor/src/store/selectors.js#L170